src/InRangeValidator.js

```diff
@@ -1,22 +1,29 @@
-const {isNil, isNumber} = require('lodash');
+const {isDate, isNil, isNumber} = require('lodash');
 const Validator = require('./Validator.js');
+const {parseDate} = require('../src/util/util');
 
 class InRangeValidator extends Validator {
     constructor (arg, arg2) {
         if (isNil(arg) || isNil(arg2)) {
             throw new Error('Both lower and upper bounds must be specified.');
         }
 
-        if (!isNumber(arg) || !isNumber(arg2)) {
-            throw new Error('Both arguments must be numbers.');
+        if ((!isNumber(arg) && !isDate(arg)) || (!isNumber(arg2) && !isDate(arg2))) {
+            throw new Error('Both arguments must be numbers or Date(s).');
         }
 
         if (arg > arg2) {
             throw new Error('Lower bound must be less or equal than upper bound.');
```

Comment:
hay un error de texto `Lower bound must be lower or equal than upper bound`. Igual `bound` mucho no me gusta.

Reply:
Boundary o Limit. Si se cambia a Limit hay que hacer más refactoreo.